### ISSUE TITLE
Wrong timestamp for RFC3339 formatting (2020-06-26T22:40:16Z)

### ISSUE THREAD

**Issue by Clivern** [araddon/dateparse#106]
**Current Behaviour**

Thanks guys for this package! i was able to parse different time formats easily but i face issue with `RFC3339` format. i can see this format in the examples `2009-08-12T22:15:09Z`, the thing is the package thinks that `2020-06-26T22:40:16Z != 2020-06-26T22:40:16+00:00` in case local timezone not UTC.

in short if i run the following code locally where my timezone is CEST, the package see that
`2020-06-26T22:40:16Z != 2020-06-26T22:40:16+00:00` while it works with go time standard library

if i run the code where timezone is UTC, it works perfect.

```golang
package main

import (
    "time"
    "fmt"

    "github.com/araddon/dateparse"
)


func main() {

    thetime, e := time.Parse(time.RFC3339, "2020-06-26T22:40:16Z") // Z means UTC+0

    if e != nil {
        panic("Can't parse time format")
    }

    epoch := thetime.Unix()

    fmt.Println(epoch) // 1593211216 (06/26/2020 @ 10:40pm (UTC))


    // Works
    thetime, e = dateparse.ParseLocal("2020-06-26T22:40:16+00:00")

    if e != nil {
        panic("Can't parse time format")
    }

    epoch = thetime.Unix()

    fmt.Println(epoch) // 1593211216 (06/26/2020 @ 10:40pm (UTC))

    // Don't Work
    thetime, e = dateparse.ParseLocal("2020-06-26T22:40:16Z") // Z means UTC+0

    if e != nil {
        panic("Can't parse time format")
    }

    epoch = thetime.Unix()

    fmt.Println(epoch) // 1593204016 (06/26/2020 @ 8:40pm (UTC))

    t := time.Now()
    zone, offset := t.Zone()
    fmt.Println(zone, offset) // CEST 7200
}
```

Here is a playground where timezone is UTC https://play.golang.org/p/aOpc22XMiwR

**Expected Behaviour**

`Z` should be considered `+00:00` based on [RFC 3339](https://tools.ietf.org/html/rfc3339)

```
Z           A suffix which, when applied to a time, denotes a UTC
                  offset of 00:00; often spoken "Zulu" from the ICAO
                  phonetic alphabet representation of the letter "Z".
```

```
Here are some examples of Internet date/time format.

      1985-04-12T23:20:50.52Z

   This represents 20 minutes and 50.52 seconds after the 23rd hour of
   April 12th, 1985 in UTC.
```

Anyway in my case i can replace `Z` with `+00:00` or use go standard library but it would be nice if that got fixed since this format is common.

**Comment by solidmetanoia** [comment]
Might be a bit late, but as I understand, then Z should always set the timezone to +00:00, correct?
Just forked it and look at the source, parseany.go:993 looks for 'Z', adding this would force it into +00:00.
```diff
case 'Z':
	p.stateTime = timeZ
	if p.seci == 0 {
		p.minlen = i - p.mini
	} else {
		p.seclen = i - p.seci
	}
+     // (Z)ulu time
+     p.loc = time.UTC
```

**Comment by Clivern** [comment]
Yes, correct @solidmetanoia

**Comment by Clivern** [comment]
PR got merged #106
